FEATURE_FLAGS.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+# Superset Feature Flags
+*This is a list of the current Superset optional features. See config.py for default values. These features can be turned on/off by setting your preferred values in superset_config.py to True/False respectively*
+
+## In Development
+*These features are considered **unfinished** and should only be used on development environments.*
+
+- **CLIENT_CACHE** *Introduces a client (browser) cache*
+- **DASHBOARD_CACHE**
+- **DASHBOARD_NATIVE_FILTERS_SET**
+- **DASHBOARD_RBAC**
+- **DISABLE_DATASET_SOURCE_EDIT**
+- **ENABLE_EXPLORE_JSON_CSRF_PROTECTION**
+- **KV_STORE**
+- **PRESTO_EXPAND_DATA**
+- **REMOVE_SLICE_LEVEL_LABEL_COLORS**
+- **SHARE_QUERIES_VIA_KV_STORE**
+- **TAGGING_SYSTEM**
+
+## In Testing
+*These features are **finished** but currently being tested. They are usable, but may still contain some bugs.*
+
+- **ALERT_REPORTS** *Alerts and reports*
```

Comment:
formatting nit: perhaps we should put a `:` between the flag and the description?

Also, there's _a lot_ of italics in this file which makes it a bit hard to read. Maybe we can get rid of all the italics?

Reply:
Thanks @etr2460 I removed all of the comments in italics and instead linked to docs, and removed the bold styling on the flags. I moved this file, INTHEWILD and CODE_OF_CONDUCT to a RESOURCES directory. lmk if that seems better!

Reply:
I _think_ that's ok, although you'll want to make sure that github doesn't expect `CODE_OF_CONDUCT.md` to be at root when they do repo checks (like what's represented here https://github.com/apache/superset/community)

Reply:
Good call. Thanks for pointing that out. I move that back out to the root.